salt/orch/update.sls

```diff
@@ -68,6 +60,14 @@ update-modules:
     - require:
```

Comment:
No really related to your PR, but I was wondering, doesn't this `sync_all` imply all the `refresh_*` we are doing before? Maybe we could remove those stages...

Reply:
I'll check if they are really duplicated and if we can remove some of those (but not on this PR). Thanks!